packages/@react-spectrum/style-macro-s1/stories/StyleMacro.stories.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+import {style} from '@react-spectrum/style-macro-s1' with {type: 'macro'};
+
+export default {
+  title: 'S1 Style Macro'
+};
+
+export function Example() {
+  return (
+    <div className={style({backgroundColor: 'orange-500', color: 'black', fontSize: 'lg', paddingX: 8, paddingY: 4, borderRadius: 'default'})()}>
```

Comment:
Do we need an example passing in variables that change, like hover, to show how that works and to see that it works?